contents/teams/surveys/objectives.mdx

```diff
@@ -1,37 +1,62 @@
-# Goal 1: Talk to customers
+# ✅ Goal 1: Mobile surveys, Everywhere
 
-We'll only succeed if we are close to our customers, let's make our relationship with them stronger.
+We’ve already got mobile surveys working in React Native and iOS, and we’ve kicked off work for Flutter and Android [tracking here](https://github.com/PostHog/posthog/issues/25085). Next step: bring everything to feature parity with Web.
 
-- Do 5 customer interviews
-- Find the top 5 reference customers and build a relationship with them
-- Do a case study with a top reference customer
+- ✅ iOS
+- ✅ React Native
+- 🛠️ Android
+- 🛠️ Flutter
 
-# Goal 2: Improve quality
+# 📈 Goal 2: Level up survey analysis
 
-There are a couple of reported bugs that were not addressed quickly, we'd like to speed up this process and fix them.
+You asked — we listened. We're making survey analysis more useful and enjoyable, starting with a brand new global analytics page.
 
-- Reduce [SLA](https://en.wikipedia.org/wiki/Service-level_agreement) to fix bugs from months to weeks
-- Reduce [paper cut](https://en.wikipedia.org/wiki/Lingchi) issues by 50%
-    - Currently around ~30 [open bugs](https://github.com/PostHog/posthog/issues?q=is%3Aopen+is%3Aissue+label%3Afeature%2Fsurveys+label%3Abug)
+- [Global analytics view](https://github.com/PostHog/posthog/issues/28418)
+- A cleaner, more useful survey responses page
+- Extra polish and UX improvements across the board
```

Comment:
i think those are kind of vague. I'd say let's be more specific here:

- [Adding better insights for rating questions to help analyze them over time](https://github.com/PostHog/posthog/issues/30150)
- [Allow selecting the metadata we want to see per survey response](https://github.com/PostHog/posthog/issues/29844)
- Make it easier to see the most important information first (Results table) and improve analysis of open text results over a long period of time.

Reply:
happy to link some issues, i believe we need to pair up with a design and think about UX hence the generic "UX improvements" since we know the page is bloated and hard to follow, and for that we cannot define tasks if not subjective

Reply:
added more links